src/views/docs-view/server.ts

```diff
@@ -23,6 +23,7 @@ import {
 
 // Local imports
 import { getProductUrlAdjuster } from './utils/product-url-adjusters'
+import { getBackToLink } from './utils/get-back-to-link'
```

Comment:
I've split out `getBackToLink` to a separate utility file, it felt a bit cleaner that way.